pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+junit_family=xunit2
```

Comment:
Could you add a comment in this file about why this is needed?

Reply:
Sure, done in 606fc20